.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -13,13 +13,14 @@
 #### Type of Improvement
 
 <!-- Choose one or more from the following:
```

Comment:
```suggestion
<!-- Choose one or more from the following, by applying the correct label(s)
```

Reply:
Those without write access cannot label the PR, so I put the onus of applying the correct label on the reviewer down below.

Reply:
got it.